modules/p2/ding_interests/ding_interests.admin.inc

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Ding interaction pane
```

Comment:
This file makes no sense to me.

1. The DocBlock is not related to the content
2. The form seems to be a partial duplicate of `ting_admin_reservable_settings()`
3. The form has little to do with `ding_interests`